client/app/auth/auth.js

```diff
@@ -1,4 +1,48 @@
-angular.module('courageousTrapeze.auth', [])
+var app = angular.module('courageousTrapeze.auth', []);
+app.controller('AuthController', ['$scope', 'Auth', function($scope, Auth){
+  // var test = Counter();
+  $scope.user = {};
+  $scope.user.username = '';
+  $scope.user.password = '';
+
+  $scope.signin = function(count){
```

Comment:
I noticed there are a couple of instances that the variable `count` shows up, but I'm not sure what it's used for. Can you please clarify?